src/Intercom/Intercom.csproj

```diff
@@ -1,24 +1,24 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net452</TargetFrameworks>
-    <ReleaseVersion>3.0.0</ReleaseVersion>
-    <PackageId>Intercom.Dotnet.Client</PackageId>
+    <TargetFrameworks>netstandard2.0;</TargetFrameworks>
+    <ReleaseVersion>3.1.0</ReleaseVersion>
+    <PackageId>Mdc.Intercom.DotNetCore</PackageId>

```

Comment:
Let's leave the package name as-is. We can identify our custom fork with a version identifier